DiffPlex/DiffBuilder/InlineDiffBuilder.cs

```diff
@@ -77,6 +80,130 @@ public static DiffPaneModel Diff(IDiffer differ, string oldText, string newText,
             return model;
         }
 
+        public static DiffPaneModel Diff(
```

Comment:
Instread of having this logic close to twice, could we instread just share the `BuildDiffPieces` logic that SideBySide uses, extract it to a common place? Then let inline do one last step to flatten it?

Reply:
Yes you are probably right. Was trying to solve it twice now.